JetUtilities/src/L2Creator.cc

```diff
@@ -43,6 +45,7 @@ L2Creator::L2Creator(CommandLine& cl) {
   mpv        = cl.getValue<bool>    ("mpv",            false);
   delphes    = cl.getValue<bool>    ("delphes",        false);
   maxFitIter = cl.getValue<int>     ("maxFitIter",        30);
+  modified_fdscb= cl.getValue<bool> ("modified_fdscb", false);
```

Comment:
@Calpas Why is it necessary to access the fit by a special name? Is the fit you want not the last one attached to the histogram? Is that just not guaranteed? It just seems to me like it's adding some complexity to a rather simple operation (i.e. now we have to make sure the names match up for this and future fits which may be implemented in the same fashion).